packages/system-api/src/appliaction.ts

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (c) 2011-2018 Genestack Limited
+ * All Rights Reserved
+ * This source code is distributed under the MIT license:
+ * https://github.com/genestack/JS-SDK/tree/master/LICENSE
+ */
+
+import {
+    IS_APPLICATION_LOADED_MESSAGE_TYPE,
+    IsApplicationLoadedMessage,
+    GetApplicationPropertiesMessage,
+    GET_APPLICATION_PROPERTIES_MESSAGE_TYPE,
+    ApplicationProperties
+} from '@genestack/interfaces';
+
+import {systemCall, systemCallSync} from './system-calls';
+
+export function loadApplication() {
```

Comment:
It is mentioned in the "Proposed API" that `loadApplication` could accept an id of any application. How can it be passed to the function?

Reply:
It is said there "it **could** accept id of any application". I just proposed that in the future, when we'll figure all the details of dynamic application loading, we could use the same API to load any application, e.g.
```js
loadApplication('vendor/app-id').then((app) => { /* use app's details or API here */ })
```
Under the hood we'll just call another handler if app id is presented, which will actually load application on the page, launch it's life cycle etc. and then, when that app will be ready it will send initial application signal that that another app is loaded.